build-system/tasks/get-zindex/index.js

```diff
@@ -115,11 +122,52 @@ async function getZindexSelectors(glob, cwd = '.') {
   return filesData;
 }
 
+/**
+ * @param {string|Array<string>} glob
+ * @param {string=} cwd
+ * @return {Object}
+ */
+async function getZindexChainsInJs(glob, cwd = '.') {
```

Comment:
qq, why i dont see any await in the body of the function, why did this have to be async?

Reply:
nevermind. is it the `tempy.write` task i assume?

Reply:
The `tempy.write` task returns a promise, but this function doesn't need to be async.